cmd/kubelet/app/options/options.go

```diff
@@ -69,6 +69,7 @@ func NewKubeletServer() *KubeletServer {
 		KubeletConfiguration: componentconfig.KubeletConfiguration{
 			Address:                     "0.0.0.0",
 			CAdvisorPort:                4194,
+			CalculateVolumeDiskUsageFrequency: unversioned.Duration{5 * time.Minute},
```

Comment:
I am fine with reducing this, however the alerting disk thresholds should allow for more than 5 minutes of runway before the disk is full to give proper reaction time.  I expect disk usage to be much less volatile than other metrics (cpu, memory, qps) and require less frequent polling.   Since we don't have automated eviction, human intervention will be required to free up disk, so 1 second vs 1 minute of latency shouldn't be much different.  Thoughts?